my_project/template_graphs.py

```diff
@@ -394,8 +399,16 @@ def wind_rose(df, title, month, hour, labels):
         df = df.loc[(df["hour"] <= end_hour) | (df["hour"] >= start_hour)]
 
     spd_colors = mapping_dictionary["wind_speed"]["color"]
+    spd_unit = mapping_dictionary["wind_speed"][si_ip]["unit"]
     spd_bins = [-1, 0.5, 1.5, 3.3, 5.5, 7.9, 10.7, 13.8, 17.1, 20.7, np.inf]
-    spd_labels = speed_labels(spd_bins, units="m/s")
```

Comment:
this conversion should not happen here nested inside a chart

Reply:
I have added a conversion function for the spd_bins.